packages/design-system/src/components/Icons/ExternalLinkIcon.tsx

```diff
@@ -5,6 +5,7 @@ import { SvgIcon, IconCommonProps } from './SvgIcon';
 const defaultProps = {
   className: '',
   viewBox: '0 0 512 512',
+  ariaHidden: false,
```

Comment:
Icons have `aria-hidden` set to true by default. When setting this prop to false, it enables certain a11y features (`aria-labelledby` and `role="img"` specifically). 

I'm making an assumption ExternalLinkIcon will always be used in context where screen readers will need to understand its presence means there's an external link on the page and **setting this component to always reveal a11y**. This change will roll out to all apps using this icon component.

Alternatively, I could set the `ariaHidden` prop on the Story itself to illustrate to others how to make this icon accessible, but that requires teams go back and update their code manually. 

I figured an opt-out approach might be warranted for this specific icon - if apps don't this icon to be visible accessibly for some unknown reason, they can overwrite this default prop with `true`. Thoughts?

Reply:
I agree. Good thinking 👍